.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: java
+
+env:
+  global:
+    - CXXFLAGS="--std=c++0x"
+
+install:
+  - buildscripts/make_dependencies.sh # build protoc into /tmp/grpc-deps
+
+script:
+  - export PATH=/tmp/grpc-deps/bin:$PATH
```

Comment:
That's fair. Could you move it to before_script, so that we don't need to specify "gradle install"?